Kickstarter-iOS/Views/Cells/SettingsTextInputCell.swift

```diff
@@ -0,0 +1,151 @@
+import Prelude
+import UIKit
+
+private struct Margin {
+  static let leftRight: CGFloat = 8
+  static let topBottom: CGFloat = 15
+}
+
+/*
+
+ Preview 1
+ 
+ +---------------------------------------+
```

Comment:
Just an example of how we can better document the view hierarchy without the Interface Builder.

This was generated using https://textik.com tool.

Reply:
I realize we currently have problems running playgrounds, but one other suggestion I can try is creating a UI only playground (under the same branch as our UI code) and populate it with some values in order for us to being able to preview this without needing to build the app.

Reply:
I have removed these comments, they didn't grow on me.